### ISSUE TITLE
adaptiveHeight is missing in Typescript definitions

### ISSUE THREAD

**Issue by geoffharcourt** [FormidableLabs/nuka-carousel#881]
# Bugs and Questions

### Prerequisites

**Feel free to delete this section if you have checked off all of the following.**

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

### Describe Your Environment

- What version of nuka-carousel are you using?
5.0.11

- What version of React are you using?
17.0.2

- What browser are you using?
n/a

### Describe the Problem

I am using `adaptiveHeight` (which was a huge help to us!), but I'm getting Typescript warnings about the prop not being part of the `CarouselProps` interface. I looked at the index.d.ts file and it is indeed missing:

```ts
export interface CarouselProps {
  /**
   * Hook to be called after a slide is changed
   * @param index Index of the current slide
   */
  afterSlide?: (index: number) => void;
```

**Expected behavior:** [What you expect to happen]
**Actual behavior:** [What actually happens]

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by geoffharcourt** [comment]
Thanks for the quick fix @ValGeorgiev!

**Comment by ValGeorgiev** [comment]
@geoffharcourt I was just going to put a message here. :D It should be fixed with v5.0.12. Thank you for raising this with us.
